http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -202,7 +202,7 @@ class ClusterMonitorActor(
         )
     } else {
       logger.info(
-        s"Cluster ${cluster.projectNameString} is not ready yet and has taken ${currTimeElapsed.toString} so far (Dataproc cluster status = $status, GCE instance statuses = ${instances
+        s"Cluster ${cluster.projectNameString} is not ready yet and has taken ${currTimeElapsed.millis.toSeconds} so far (Dataproc cluster status = $status, GCE instance statuses = ${instances
```

Comment:
shall we add `seconds` after the variable? just so it's clear in the log what the unit is